src/stk/molecular/topology_graphs/host_guest/complex.py

```diff
@@ -90,10 +114,7 @@ class Complex(TopologyGraph):
     def __init__(
         self,
         host,
-        guest,
-        guest_start=None,
-        guest_target=None,
-        displacement=(0, 0, 0),
+        guests,
```

Comment:
by removing the `guest_start` etc. params, the changes here a backward incompatible. Is that intentional?

Reply:
Nope, they should be added back in.